Source/WebCore/page/csp/ContentSecurityPolicyDirectiveList.cpp

```diff
@@ -622,11 +622,12 @@ void ContentSecurityPolicyDirectiveList::parseRequireTrustedTypesFor(ParsedDirec
             }
 
             auto begin = buffer.position();
-            if (skipExactlyIgnoringASCIICase(buffer, "'script'"_s))
+            if (skipExactlyIgnoringASCIICase(buffer, "'script'"_s) && (buffer.atEnd() || isUnicodeCompatibleASCIIWhitespace(*buffer)))
```

Comment:
This is the wrong kind of whitespace. Though maybe we should fix that as part of #13197. Would it be easy to add whitespace tests?